sprint9-objects/ex1.js

```diff
@@ -0,0 +1,26 @@
+// 1. Scrieți o funcție care creează un obiect pentru a păstra informații despre rețeta ta preferată
+
+// Ar trebui să aibă proprietăți pentru titlu (un șir de caractere), porții (un număr) și ingrediente (o șir de elemente - array). Va fi afișată pe rânduri separate (o instrucțiune console.log pentru fiecare) toate informațiile obiectului după crearea acestuia.
+
+// Exemplu 1:
+
+// Input: titlu = “Pizza”, portii = 3, ingrediente = [“sunca”, “cas”, “ketchup”]
+
+// Output: { titlu: “Pizza”, portii: 3, ingrediente: [”sunca”, “cas”, “ketchup”] }
+
+function getReteta (titlu,portii,ingrediente) {
+    const myReteta = {
+        titlu : titlu, 
+        portii : portii, 
+        ingrediente : ingrediente
```

Comment:
```suggestion
        ingrediente : [... ingrediente],
```

cand copiem variabile care nu sunt primitive (precum liste sau obiecte), e bine sa ne amintim sa le si clonam, astfel incat noua valoare sa aiba propria sa proprietate, dar nu pointer (referinta) catre cealalta primita.